grove/blackbox_algorithms/tests/test_utils.py

```diff
@@ -0,0 +1,153 @@
+"""Test class for methods found in the black box utils."""
+
+from grove.blackbox_algorithms.utils import *
+import pyquil.quil as pq
+from pyquil.resource_manager import Qubit
+import unittest
+
+
+class TestUnitaryFromFunction(unittest.TestCase):
```

Comment:
The rest of Grove/Pyquil uses `pytest` rather than `unittest`. It would be much better if we could keep all of our codebase on the same testing package. Can you refactor these to use `py.test` instead?